pkg/controller/operators/catalog/operator.go

```diff
@@ -114,6 +125,7 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 		logger:                 logger,
 		clock:                  clock,
 		opClient:               opClient,
+		dynamicClient:          dynamicClient,
```

Comment:
👍 as a bonus, this will let us support "any" bundle object in the future and clean up ExecutePlan